.github/workflows/tapioca-update.yml

```
@@ -0,0 +1,38 @@
+name: Tapioca Update
+on:
+  pull_request:
+    paths:
+      - .github/workflows/tapioca-update.yml
+  schedule:
+    - cron: '0 0 */3 * *' # Every three days
+env:
+  HOMEBREW_GITHUB_ACTIONS: 1
+  HOMEBREW_SORBET: 1
+jobs:
+  update-tapioca:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macOS-latest]
```

```suggestion
    runs-on: ubuntu-latest
```

Unless it needs macOS in which case:

```suggestion
    runs-on: macOS-latest
```

(Linux is preferable because it's much faster)